rebuild.sh

```diff
@@ -122,6 +122,14 @@ mvnBuildDocker() {
     *)
       mvnImage=maven:${mvnVersion}-eclipse-temurin-${jdk}-alpine
   esac
+  if ! docker pull -q ${mvnImage} > /dev/null 2>&1; then
+    for image in maven:{${mvnVersion},3}-eclipse-temurin-${jdk}-alpine; do
+      if docker pull -q ${image} > /dev/null 2>&1; then
+        mvnImage=${image}
+        break
+      fi
```

Comment:
I realized that the previous revision could inspect more images than necessary. Are simple for-loop avoids that.